daffodil-japi/src/main/scala/org/apache/daffodil/japi/infoset/Infoset.scala

```diff
@@ -40,6 +40,8 @@ import org.apache.daffodil.infoset.DIComplex
 import org.apache.daffodil.infoset.DIArray
 import org.apache.daffodil.dpath.NodeInfo
 
+import org.apache.daffodil.api.XMLOutputStyle
```

Comment:
I'm not sure if Java users can easily use this enum. See what we do with the japi for ValidationMode and ValidationConversions as a pattern for how to handle enums.